custom_components/tuya_local/devices/loratap_zigbee_six_switch.yaml

```diff
@@ -0,0 +1,232 @@
+name: Loratap Six Gang ZigBee Remote Switch
```

Comment:
Move this full name into the products section, and use something more generic like "Six button remote" here.  It is possible that this device will be sold under different brands in different markets, or that other similar devices will use the same basic dp layout.

Also, HA's naming convention is that only the first word, trademarks and abbreviations should be capitalised in names.